src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/Referral.kt

```diff
@@ -5,6 +5,8 @@ import org.hibernate.annotations.FetchMode
 import org.hibernate.annotations.FetchMode.JOIN
 import java.time.LocalDate
 import java.time.OffsetDateTime
+import java.util.Objects.isNull
```

Comment:
usages of these seem kinda non-kotlin-y

Reply:
IMO i think this reads clearer as just `concludedAt != null && endRequestedAt !=null && endOfServiceReport == null`

Reply:
this means referrals with EOSRs still stick around forever?

Reply:
for now, in this dashboard, yes.
I expect there'll be filtering options to the endpoint later on to support the "live" and "ended" dashboards

Reply:
moved to `null` equality checks in 514cb097